datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -0,0 +1,278 @@
+package datadog
+
+import (
+	"errors"
+	"fmt"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceDatadogSecurityMonitoringDefaultRule() *schema.Resource {
+	return &schema.Resource{
+		Exists: resourceDatadogSecurityMonitoringDefaultRuleExists,
```

Comment:
You can remove exist method, this is not useful in the end.

Reply:
Removed